src/AppEntry.tsx

```
@@ -1,7 +1,6 @@
 import React from 'react';
-
 import App from './App';
 
-const AppEntry = (): JSX.Element => <App />;
+const AppEntry = () => <App />;
 
-export default AppEntry;
+export default AppEntry;
```

ESLint

[error] Insert `⏎`

(prettier/prettier)